netman/dbus.go

```diff
@@ -135,104 +171,111 @@ func getSystemBus() *dbus.Conn {
 	return conn
 }
 
-// Ssids returns an array of available ssids
-func Ssids() ([]SSID, map[string]string, map[string]string) {
+func (c *Client) Ssids() ([]SSID, map[string]string, map[string]string) {
 	ap2device := make(map[string]string)
 	ssid2ap := make(map[string]string)
-
-	devices := getDevices()
-	wifiDevices := getWifiDevices(devices)
-	APs := getAccessPoints(wifiDevices, ap2device)
-	SSIDs := getSSIDs(APs, ssid2ap)
+	devices := c.GetDevices()
+	wifiDevices := c.GetWifiDevices(devices)
+	APs := c.GetAccessPoints(wifiDevices, ap2device)
+	SSIDs := c.GetSsids(APs, ssid2ap)
 	return SSIDs, ap2device, ssid2ap
 }
 
-// ConnectedWifi returns true if connected to network by a netman interface
-func ConnectedWifi() bool {
-	conn := getSystemBus()
-	objPath := dbus.ObjectPath("/org/freedesktop/NetworkManager")
-	nm := conn.Object("org.freedesktop.NetworkManager", objPath)
-	var nmConnectivityStatus uint32
-	err := nm.Call("org.freedesktop.NetworkManager.CheckConnectivity", 0).Store(&nmConnectivityStatus)
-	if err != nil {
-		fmt.Printf("Error in ConnectedWifi(): %v\n", err)
-		return false
-	}
-	if nmConnectivityStatus == 1 {
-		return false //not connected by any netman interfaces
+// check if no ethernet or wifi devices are connected
```

Comment:
comment on exported method Client.Connected should be of the form "// Connected ..."